Description: Disable /dev/tty needing test in test-suite
 One test in C02cond.ztst of the upstream test-suite always fails if
 /dev/tty does not exist or is no character device..
 .
 This may be the case on the GNU/Hurd buildds.
 .
 This patch removes that test. As this is not always wanted, this test
 should never be listed in debian/patches/series, but will be applied
 when necessary by debian/patch-test-suite.sh.
 .
 Again: DO NOT LIST THIS PATCH IN THE series FILE.
Author: Axel Beckert <abe@debian.org>

Index: zsh/Test/C02cond.ztst
===================================================================
--- zsh.orig/Test/C02cond.ztst	2013-09-14 02:42:12.000000000 +0200
+++ zsh/Test/C02cond.ztst	2013-09-20 16:04:20.000000000 +0200
@@ -41,11 +41,6 @@
   fi
 0D:-b cond
 
-  # Use hardcoded /dev/tty because globbing inside /dev fails on Cygwin
-  char=/dev/tty
-  [[ -c $char && ! -c $zerolength ]]
-0:-c cond
-
   [[ -d . && ! -d zerolength ]]
 0:-d cond
 
